src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_copy_b.hpp

```diff
@@ -42,8 +42,11 @@ class BrgemmCopyB : public snippets::op::MemoryAccess {
     size_t get_n_block_size() const { return m_N_blk; }
     void set_k_block_size(size_t block_size) { m_K_blk = block_size; }
     void set_n_block_size(size_t block_size) { m_N_blk = block_size; }
+    size_t get_k_rounded() const { return m_K_rounded; }
+    size_t get_n_rounded() const { return m_N_rounded; }
```

Comment:
Do we really these setters? To be honest for me `_rounded` sounds strange 😄 
I'd suggest to create two direct methods `get_data_repacking_shape` and `get_compensation_shape` at least.
Could you please verify that it's more comfortable than using these getters?
Moreover, as I understand these class fields are depended on shapes. It's not very cool to store them in class so when we are working on dynamism support.

Reply:
Agree with you. The mentioned methods are created and reused in `SetBrgemmCopyBBuffersShape` transformation